src/commands/suggest.ts

```diff
@@ -45,7 +52,7 @@ export class UserCommand extends Command {
 			.setDefaultMemberPermissions(PermissionFlagsBits.ManageMessages)
 			.setDMPermission(false)
 	)
-	public messageContextRun(interaction: Command.MessageInteraction, options: TransformedArguments.Message) {
+	public messageContextRun(interaction: Command.MessageInteraction, options: TransformedArguments.Message): Promise<unknown> {
```

Comment:
```suggestion
	public messageContextRun(interaction: Command.MessageInteraction, options: TransformedArguments.Message) {
```

No longer needed thanks to my fix for HFx